api_docs/incoming-webhooks-overview.md

```diff
@@ -197,3 +197,7 @@ Parameters accepted in the URL include:
 [browse-streams]: /help/browse-and-subscribe-to-streams
 [add-bot]: /help/add-a-bot-or-integration
 [url-encoder]: https://www.urlencoder.org/
+
+## Related articles
+
+* [Integrations overview](/api/integrations-overview)
```

Comment:
Even though it's linked in the article, I'd link to the example incoming webhook walk through here as well. And maybe the non-webhook integrations article as well.